tests/PySys/software-management-end-to-end/environment_sm_management.py

```diff
@@ -20,17 +20,44 @@
 
 
 
+TODO: Avoid hardcoded ids
+TODO: Get software package ids from c8y
+TODO: Add management for package creation and removal for c8y
+    -> Mabe as separte python module to access c8y
```

Comment:
```suggestion
    -> Maybe as separate python module to access c8y
```

Reply:
I've tried to introduce a similar `Cumulocity` interface for all Cumulocity operations here: https://github.com/thin-edge/thin-edge.io/pull/416/files#diff-b189e81ee390bd839972c43fd8483b4814846e68e9bc8e223dd38c2974205543R14

It can be enhanced with all this software repository management, as well as software update actions as well.